lib/CXGN/Trial/TrialDesign.pm

```diff
@@ -312,6 +390,26 @@ sub _get_rcbd_design {
   } else {
     die "Number of blocks not specified\n";
   }
+  sub isint{
```

Comment:
I would have to move it out of each of the design function and have it as a separate function.